src/site/content/en/blog/responsiveness/index.md

```diff
@@ -0,0 +1,304 @@
+the overall responsiveness of a page throughout its lifetime.
+
+We've made a lot of progress on this metric in the last few months, and we
+wanted to share an update on how we plan to measure interaction latency as well
+as introduce a few specific aggregation options we're considering to quantify
+the overall responsiveness of a web page.
+
+We'd love to get [feedback](#feedback) from developers and site owners as to
+which of these options would be most representative of the overall input
+responsiveness of their pages.
+
+## Measure interaction latency
+
+As a review, the [First Input Delay (FID) metric captures the
+delay portion](/fid/#fid-in-detail) of input latency. That is, the time between
+when the user interacts with the page to the time when the event handlers are
+able to run.
+
+With this new metric we plan to expand that to capture the [full event
+duration](/better-responsiveness-metric/#capture-the-full-event-duration), from
+initial user input until the next frame is painted after all the event handlers
+have run.
+
+We also plan to measure
+[interactions](/better-responsiveness-metric/#group-events-into-interactions)
+rather than individual events. Interactions are groups of events that are
+dispatched as part of the same, logical user gesture (for example:
+`pointerdown`, `click`, `pointerup`).
+
+To measure the total interaction latency from a group of individual event
+durations, we are considering two potential approaches:
+
+- **Maximum event duration:** the interaction latency is equal to the largest
+  single event duration from any event in the interaction group.
+- **Total event duration:** the interaction latency is the sum of all event
+  durations, ignoring any overlap.
+
+As an example, the diagram below shows a _key press_ interaction that consists
+of a `keydown` and a `keyup` event. In this example there is a duration overlap
+between these two events. To measure the latency of the key press interaction,
+we could use `max(keydown duration, keyup duration)` or `sum(keydown duration,
+keyup duration) - duration overlap`:
+
+{% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/td8jMQBTjyR34ZO1QBvp.svg", alt="A
+diagram showing interaction latency based on event durations", width="800",
+height="424" %}
+
+There are pros and cons of each approach, and we'd like to collect more data and
+[feedback](#feedback) before finalizing a latency definition.
+
+{% Aside %}
+  The event duration is meant to be the time from the event hardware timestamp
+  to the time when the next paint is performed after the event is handled. But
+  if the event doesn't cause any update, the duration will be the time from
+  event hardware timestamp to the time when we are sure it will not cause any
+  update.
+{% endAside %}
+
+{% Aside %}
+  For keyboard interactions, we usually measure the `keydown` and `keyup`. But
+  for IME, such as input methods for Chinese and Japanese, we measure the input
```

Comment:
the input event here is https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event .
maybe change it to blue as well ?

Reply:
Updated.